src/Reflection/ReflectionObject.php

```diff
@@ -95,9 +95,10 @@ public static function createFromInstance($object)
     /**
      * Reflect on runtime properties for the current instance
      *
+     * @param ?int $filter
```

Comment:
would it be ok to point directly to [http://php.net/manual/en/reflectionclass.getmethods.php](http://php.net/manual/en/reflectionclass.getmethods.php) where the parameter meaning is explained?